src/Files.Uwp/Views/MainPage.xaml

```diff
@@ -222,7 +222,6 @@
         HorizontalAlignment="Stretch"
         HorizontalContentAlignment="Stretch"
         Background="{ThemeResource RootBackgroundBrush}"
-        CanOpenInNewPane="{x:Bind SidebarAdaptiveViewModel.PaneHolder.IsMultiPaneEnabled, Mode=OneWay}"
```

Comment:
Was this removed on purpose?

Reply:
Yes, this variable was used only to set context menu item visibility. 
Should I restore it to the global scope of the SidebarControl.xaml.cs file?

Reply:
It should be kept in the global scope since the sidebar isn't supposed to have access to this information on its own.

Reply:
Ok

Reply:
Done!